src/www/diag_confbak.php

```diff
@@ -226,7 +226,7 @@
                   </td>
                   <td>
                     <?= gettext('Be aware of how much space is consumed by backups before adjusting this value.'); ?>
-                    <?php if (count($confvers)) {
+                    <?php if (isset($confvers) && count($confvers) > 0) {
```

Comment:
been over this recently and it's set in POST and else branch without an early way to skip so it looks like a false positive. Ideally I'd want this to break if it fails after a rework